src/Service/WpCronService.php

```diff
@@ -32,8 +32,23 @@ public function dispatch($callback, ...$args): void
      */
     public function schedule($callback, int $timestamp, ...$args): void
     {
+        if (! is_string($callback) && ! is_array($callback)) {
+            throw new InvalidArgumentException('Invalid callback');
+        }
+
         $hook = $callback;
 
+        if (is_array($callback)) {
```

Comment:
Dit voelt niet heel logisch:

- Wanneer callback een array is wordt deze direct uitgevoerd en niet gescheduled. Maar ik zie daar niet echt heldere documentatie/comments bij waarom dat zo zou moeten zijn?
- Ik zou verwachten dat er ook een Exception is wanneer de `$callback` een niet-bestaande class (is al het geval) of method (is nog niet het geval) bevat. Nu faalt dat stilletjes en heb je niets door?

Reply:
- Comment toegevoegd
- Het faalt niet, hij gaat eroverheen (omdat ie niet returned) en scheduled de task gewoon, zoals voorheen (wat dus niet werkt, maar zo heb ik de functionaliteit exact gelijk gehouden, behalve dat ik de dingen die wel kunnen werken, zoals de webhooks, eruit pik)

Reply:
Maar als dat schedulen niet _kan_ werken... Het is niet iets wat ooit "gefixt" gaat worden ofzo, het is toch gewoon hoe het in wp werkt? Ik vind het niet echt een heel heldere oplossing zo :/

Reply:
Er zijn twee redenen waarom het niet kan werken:
1) een instantiated class kan niet uitgevoerd worden, daar is een facade voor nodig in de pdk denk ik, om die aan te roepen en de class dan te instantiaten
2) de wp cron service is _onbetrouwbaar_, ik verwacht eigenlijk dat daar nog wel iets aan gedaan kan / zal worden, hij is al meer betrouwbaar als we een andere timestamp gebruiken die verder in het verleden ligt (al haalt die ook dan nog niet 100%)

Reply:
Ik probeerde eigenlijk zo snel mogelijk een betrouwbare oplossing te maken zodat klanten de webhooks kunnen gebruiken.